wikidict/utils.py

```diff
@@ -420,9 +420,9 @@ def process_templates(word: str, text: str, locale: str) -> str:
 
         >>> process_templates("octonion", " <math>V^n</math>", "fr")  # doctest: +ELLIPSIS
         '<img style="height:100%;max-height:0.8em;width:auto;vertical-align:bottom" src="data:image/gif;base64,...'
-        >>> process_templates("test", r"<math>\R^n</math>", "fr")
-        <math> ERROR with \R^n in [test]
-        '\\R^n'
+        >>> process_templates("test", r"<math>\frac</math>", "fr")
```

Comment:
With that PR, there should be no more error, right?

Reply:
As explained in the issue,  `\frac` is not supported and gives an error.

Reply:
Ah indeed :)